### DISCUSSION TITLE
Error en la entrega de la tarea 4 laboratorio 04.01

### DISCUSSION THREAD

**Discussion by Apolo4427** [rramosp/ai4eng.v1#284] [04 - GESTIÓN DE DATOS]
Buenas tardes, estoy elaborando el laboratorio 04.01 y me sale este error, no lo comprendo
por que parece que no existiera la columna "place" sin embargo en el archivo  "risk.csv" si aparece dicha columna, no se que debería hacer ahí.
![image](https://github.com/rramosp/ai4eng.v1/assets/118028519/71aab55b-12f5-44f6-a5b1-46e106b34354)
El problema, además solo aparece al momento de subir la respuesta, puesto que en el notebook funciona con normalidad, me reemplaza la columna "place":
![image](https://github.com/rramosp/ai4eng.v1/assets/118028519/2d87d169-3630-4fb4-be56-e96dc6a2b1ad)

**Comment by Apolo4427** [comment]
Ya hallé el error, la clave era 'lugar', no 'place'.

**Reply by Felipe-RA** [reply]
Gracias por compartirlo con nosotros! @Apolo4427
